src/data/faq.json

```diff
@@ -2171,66 +2171,299 @@
+                },
+                {
+                    "term": "Error report",
+                    "anchor": "error_report",
+                    "description": "A detailed report of a malfunction that has occurred, which is sent to the responsible developers so that they can fix the described error."
+                },
                 {
                     "term": "EU DCC",
                     "anchor": "eu_dcc",
                     "description": "See <a href='#glossary_dcc'>DCC</a>"
+                },
+                {
+                    "term": "European Federation Gateway Service (EFGS)",
+                    "anchor": "efgs",
+                    "description": "A central infrastructure that is used to exchange and check <a href='#glossary_diagnosis_key'>diagnosis keys</a> across national borders."
+                },
+                {
+                    "term": "Event registration",
+                    "anchor": "event_registration",
+                    "description": "Register for an event or a visit to a location."
+                },
+                {
+                    "term": "Exposure Detection",
+                    "anchor": "exposure_detection",
+                    "description": "List of short-lived random Bluetooth IDs received and temporarily stored in the operating system memory. The encounter record is read as part of the risk determination during the risk check. All short-lived random Bluetooth IDs older than 14 days are automatically deleted. The encounter record can also be deleted in its entirety at any time by individuals using the app. When the CWA is uninstalled, the encounter record is automatically deleted."
+                },
+                {
+                    "term": "Exposure Notification Express (ENE)",
+                    "anchor": "ene",
+                    "description": "See FAQ: <a href='https://www.coronawarn.app/en/faq/#ene' target='_blank' rel='noopener noreferrer'>Exposure Notification Express (ENE)</a>"
+                },
+                {
+                    "term": "Exposure Notification Framework (ENF)",
+                    "anchor": "enf",
+                    "description": "The Google Exposure Notification Framework is a developed OS-level system whose goal is to reduce the spread of COVID-19 through contact tracing. It enables the exchange of random keys via Bluetooth Low Energy (BLE)."

```

Comment:
This section needs to contain the translation from the German text: "Das Exposure Notification Framework (Apple Terminologie) bzw. Exposure Notification System (Google Terminologie) ist ein von Google und Apple entwickeltes System auf Betriebssystemebene, dessen Ziel die Reduzierung der Verbreitung von COVID-19 durch Contact-Tracing ist. Es ermöglicht den Austausch von Zufallsschlüsseln über Bluetooth Low Energy (BLE)."

Reply:
... followed by the entry "Exposure Notification System (ENS)" like in the German version.

Reply:
Will be solved in the next commit, thanks!